quinn-proto/src/connection/mod.rs

```diff
@@ -3058,6 +3103,53 @@ impl Connection {
                 self.stats.frame_tx.handshake_done.saturating_add(1);
         }
 
+        // OBSERVED_ADDR
+        let mut send_observed_address =
```

Comment:
This name should reflect that it might not do anything.
```suggestion
        let mut maybe_send_observed_address =
```